tests/installation/installer_desktopselection.pm

```diff
@@ -16,8 +16,11 @@ sub run() {
     assert_screen 'desktop-selection';
     my $d = get_var('DESKTOP_MINIMALX_INSTONLY') ? 'minimalx' : get_var('DESKTOP');
     if ($d ne 'kde' && $d ne 'gnome') {
-        send_key_until_needlematch 'selection_on_desktop_other', 'tab';    # Move the selection to 'Other'
-        send_key 'spc';                                                    # open 'Other' selection'
+        # up to 42.1 textmode was below 'other'
+        if (!($d eq 'textmode' && check_screen 'has-server-selection')) {
```

Comment:
Maybe just do a static version check instead of the check screen? Also saves 30 seconds on non-match. Or adjust the previous assert screen to match on multiple tags and use match_has_tag?